support-frontend/assets/pages/paper-subscription-landing/components/hero/discountCopy.jsx

```diff
@@ -14,11 +14,11 @@ const discountCopy = (discountPercentage: number): DiscountCopy => (
   discountPercentage > 0 ?
     {
       roundel: ['Save up to', `${discountPercentage}%`, 'a year'],
-      heading: `Save up to ${discountPercentage}% a year with a subscription`,
+      heading: [`Save up to ${discountPercentage}% for a year on The Guardian`, <br />, ' and The Observer'],
```

Comment:
This should just be 'a year ...' not 'for a year' because it is not an introductory offer